src/main/java/net/pms/util/BitRateMode.java

```diff
@@ -0,0 +1,52 @@
+package net.pms.util;
+
+import static org.apache.commons.lang3.StringUtils.isBlank;
+import java.util.Locale;
+
+/**
+ * This {@code enum} represents the possible bit rate "modes".
+ */
+public enum BitRateMode {
+
+	/** Constant bit rate mode*/
+	CBR,
+
+	/** Variable bit rate mode */
+	VBR;
+
+	@Override
+	public String toString() {
+		switch (this) {
+			case CBR:
+				return "Constant";
```

Comment:
@Sami32 You need a bigger computer screen. I have 3 screens, so I don't need everything to be in cryptic codes 😛 

I see your point though and it's some truth to it, but I personally find ```CBR``` and ```VBR``` to be too similar so that I often misread them. IMO, ```C``` and ```V``` would be better. It might just be my eyes, but they do have some problems separating the two.